README.md

```diff
@@ -33,7 +33,7 @@ The libraries released in July preview:
 - [azure-keyvault-secrets](./sdk/keyvault/azure-keyvault-secrets)
 - [azure-identity](./sdk/identity/azure-identity)
 - [azure-storage-blob](./sdk/storage/azure-storage-blob)
-- [azure-storage-file](./sdk/storage/azure-storage-file)
+- [azure-storage-file](./sdk/storage/azure-storage-file-share)
```

Comment:
Should the link text be changed too?

Reply:
I don't think so - this section in the readme is for the historical (July preview) `azure-storage-files` release. I just didn't want the link to point to a 404 so I redirected it to the new equivalent.